### PR TITLE
#29: migration from js to ts.

### PR DESCRIPTION
**Pull request by thebishaldeb**
#29 run yarn install to update the depndencies

### CODE REVIEW

**Comment by thebishaldeb** [comment]
@armujahid Done!

File: src/utils/Api.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
```
Comment (armujahid) [inline src/utils/Api.tsx]: Kindly change its extension to ts

**Comment by armujahid** [comment]
Thanks. Looks nice. I will check it soon.

File: src/components/CustomLoader.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ContentLoader from "react-content-loader"
 import { Card } from 'reactstrap';
 
-const CustomLoader = props => (
+const CustomLoader = (props: any) => (
```
Comment (armujahid) [inline src/components/CustomLoader.tsx:5]: I will create a separate issue for types. No worries

**Review by armujahid — CHANGES_REQUESTED** [review]

File: src/utils/Api.tsx (hunk 1 above)
Comment (thebishaldeb) [reply src/utils/Api.tsx]: Done!

File: src/components/CustomLoader.tsx (hunk 2 above)
Comment (thebishaldeb) [reply src/components/CustomLoader.tsx:5]: Cool!

**Comment by thebishaldeb** [comment]
Check it out now @armujahid

**Comment by armujahid** [comment]
Thanks @thebishaldeb. I merged it in a different branch so that other improvements can be done before merging it in master :)

**Pull request merged**